code/game/gamemodes/objective.dm

```diff
@@ -5,6 +5,7 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 	var/datum/team/team //An alternative to 'owner': a team. Use this when writing new code.
 	var/name = "generic objective" //Name for admin prompts
 	var/explanation_text = "Nothing" //What that person is supposed to do.
+	var/no_failure = FALSE //if this objective doesn't print failure or success
```

Comment:
```suggestion
	/// if this objective doesn't print failure or success in the roundend report
	var/no_failure = FALSE 
```
Everything else her isn't auto docced but we may as well be. Also a small clarification.